src/Home/index.js

```diff
@@ -178,7 +178,7 @@ export const Home = () => (
 									title="The blockchain will disrupt the music business and beyond"
 								>
 									<img
-										src="https://www.taittowers.com/wp-content/uploads/2016/07/WORKS_wired-magazine.jpg"
+										src="https://www.musicoin.org/images/logos/wired.jpg"
```

Comment:
do we have this image in the backend?

Reply:
Yes, I have uploaded it into public.

Reply:
thanks, but this public is not same as the other public, but I can move it myself.